internal/pubsub/rabbitpubsub/rabbit.go

```diff
@@ -184,6 +184,17 @@ func (*topic) IsRetryable(error) bool {
 	return false
 }
 
+// As implements driver.Topic.As.
```

Comment:
Same: document the types supported by the various `As` functions in the package godoc.